app/helpers/admin/analytics_helper.rb

```diff
@@ -8,5 +8,22 @@ def track_analytics_data(category, action, label)
         "track-label" => ActionController::Base.helpers.strip_tags(label),
       }
     end
+
+    def track_analytics_data_on_load(title)
+      {
+        event_name: "page_view",
+        page_view: {
+          government_department_name: "Government Digital Service",
+          access_level: access_based_roles(current_user&.permissions),
+          publishing_app: "Whitehall",
+          experience: current_user&.created_at,
+          document_type: title,
+        },
+      }.to_json
+    end
+  end
+
+  def access_based_roles(permissions)
```

Comment:
Was just in a chat with Carmen and another PA and it seems there is a character limit on GTM, so perhaps it's not ideal to send the full role array + most of the roles aren't exactly app-access related

Reply:
spoke about it - a separate list to maintain which permissions to exclude sounds like effort to maintain for not a lot of value - leave it as is and see if it causes any issues.Most users won't have all the permissions